src/intrprtr.c

```diff
@@ -525,7 +525,7 @@ void IntrFuncExprEnd(UInt nr)
     assert(STATE(IntrCoding) > 0);
 
     STATE(IntrCoding)--;
-    CodeFuncExprEnd(nr);
+    CodeFuncExprEnd(nr,0);
```

Comment:
Based on the name `useStack`, I'd actually expect this:
```suggestion
    CodeFuncExprEnd(nr, 1);
```